src/client/services/calendar/calendar.js

```diff
@@ -0,0 +1,85 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ */
+
+'use strict';
+
+var assign = require('lodash.assign');
+var defaults = require('lodash.defaults');
+var isArray = require('lodash.isarray');
+var noop = require('lodash.noop');
+var patterns = require('../../../util/patterns');
+var resolveWith = require('../../../util/resolve-with');
+var SparkBase = require('../../../lib/spark-base');
+
+// Adding decryption for subject
+require('./decrypter');
+
+/**
+ * @class
+ * @extends {SparkBase}
+ * @memberof Calendar
+ */
+var CalendarService = SparkBase.extend(
+  /** @lends Calendar.CalendarService.prototype */
+  {
+  namespace: 'Calendar',
+
+  /**
+   * Retrieves a collection of meetings based on the query parameters
+   *
+   * @memberof CalendarService.prototype
+   * @param {Date} start the start of the time range
+   * @param {Date} end the end of the time range
+   * @returns {Promise} Resolves with an array of meetings
+   */
+  getMeetings: function getMeetings() {
+    return this.request({
+      api: 'calendar',
+      resource: 'calendarEvents'
+    })
+    .then((res) => {
+      if (res.body.items) {
+        return Promise.all(
+          res.body.items.map((meeting) => {
+            meeting.title = meeting.encryptedSubject;
+            return this.spark.conversation.decrypter.decryptProperty(meeting, 'title', meeting.encryptionKeyUrl);
+          })
+        ).then(
+          // At this point the decryption has been finished
+          function processDecryptedData(data) {
+            console.log(data);
+            return data;
+          }
+        ).catch((err) => {
+          this.logger.error('calendar: error occurred in decrypting meeting', err);
+        });
+      }
+      else {
+        return [];
+      }
+    })
+    .catch((err) => {
+      this.logger.error('calendar: error occurred in retrieving meetings', err);
+    });
+  },
+
+  getUser: function getUser(userId, orgId) {
+    const CISCO_ORG_ID = '1eb65fdf-9643-417f-9974-ad72cae0e10f';
+    orgId = orgId || CISCO_ORG_ID;
+
+    return this.request({
+      method: 'GET',
+      url: `https://identity.webex.com/identity/scim/${orgId}/v1/Users/${userId}`
```

Comment:
The data that is retrieved in the call to getMeetings only has the participants' UUID's. As a result, to populate the data in the web client, we need to retrieve user data from their UUID. Unfortunately, we can't use the endpoint in Conversation because the meetings don't have any conversation context, so I was forced to end up talking to SCIM to retrieve user information.